packages/remme-web-socket/package.json

```diff
@@ -11,9 +11,12 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "websocket": "^1.0.26"
+    "websocket": "^1.0.26",
+    "remme-utils": "^0.5.0",
+    "remme-atomic-swap": "^0.5.2"
```

Comment:
don't need, because we should provide SwapInfo model from websocket to atomic swap package.
In future we provide all models to one place maybe remme-models or remme-utils package